server/routes/github/github_app.py

```
@@ -0,0 +1,1028 @@
+            return jsonify({"error": "Failed to reach GitHub"}), 502
+
+        if resp.status_code != 200:
+            logger.error(
+                "[GITHUB-APP-DISCOVER] GitHub returned status=%d", resp.status_code
+            )
+            return jsonify({"error": "Failed to list App installations"}), 502
+
+        try:
+            page = resp.json()
+        except ValueError:
+            logger.error("[GITHUB-APP-DISCOVER] response not JSON")
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)